Emby.Server.Implementations/LiveTv/TunerHosts/SharedHttpStream.cs

```diff
@@ -94,7 +94,7 @@ public override async Task Open(CancellationToken openCancellationToken)
 
             var now = DateTime.UtcNow;
 
-            var _ = StartStreaming(response, taskCompletionSource, LiveStreamCancellationTokenSource.Token);
+            _ = StartStreaming(response, taskCompletionSource, LiveStreamCancellationTokenSource.Token);
```

Comment:
Would this work without the assignment?

Reply:
You’ll get a warning, that what the discard syntax is for.